src/components/DataArchives/DataArchives.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: 'white'
+  },
+  main: {
+    paddingBottom: '3rem',
+    [theme.breakpoints.up('xs')]: {
+      paddingRight: '3rem',
+      paddingLeft: '3rem'
+    }
+  },
+  link: { color: theme.palette.primary.dark },
+  typography: {
+    paddingTop: theme.spacing.unit * 6,
+    textAlign: 'center',
+    fontWeight: 'bold'
+  },
+  title: {
+    fontWeight: 700,
+    fontSize: '1rem'
+  },
+  archive: { paddingTop: '2rem', paddingBottom: '2rem' },
+  code: {
+    boxSizing: 'border-box',
+    display: 'inline-block',
+    background: '#fafcfc',
+    padding: '12px 10px',
+    border: '1px solid #f0f4f7',
+    borderRadius: '4px',
+    lineHeight: '20px',
+    fontSize: '14px'
+  },
+  var: {
+    color: theme.palette.primary.dark
+  }
+});
+
+const now = 'http://api.airquality.codeforafrica.org/v1/now/';
+const data = 'http://api.airquality.codeforafrica.org/v1/data/';
+const sensors = 'http://api.airquality.codeforafrica.org/v1/sensor/{apiID}/';
+const query = 'http://api.airquality.codeforafrica.org/v1/filter/{query}';
+const type = '{ sensor type }';
+const area = '{ lat, lon, distance }';
+const box = '{ lat1, lon1, lat2, lon2 }';
+const countryCode = '{ country code }';
+
+function DataArchives({ classes }) {
+  return (
+    <Grid
+      container
+      justify="center"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid
+        item
+        xs={12}
+        sm={6}
+        justify="center"
+        align="left"
+        className={classes.main}
+      >
+        <Typography variant="h6" className={classes.typography}>
+          Sensors Data
```

Comment:
Everything on this page is talking about accessing the data, shouldn't this heading be **Access Sensors Data**?